libs/net/traffic_generator.py

```diff
@@ -87,16 +87,14 @@ class VMTcpClient(BaseTcpClient):
         vm (BaseVirtualMachine): The virtual machine where the client runs.
         server_ip (str): The destination IP address of the server the client connects to.
         server_port (int): The port on which the server listens for connections.
+        maximum_segment_size (int):MTU customization (by customizing mss) in bytes.
```

Comment:
This text is confusing. The MSS is the TCP payload size. Many times it is calculated from the MTU by subtracting the IP & TCP header sizes, but that is not relevant in this context.

I suggest you just say something like:
> Define explicitly the TCP payload size (in bytes).